packages/lesswrong/components/common/AnalyticsTracker.jsx

```diff
@@ -0,0 +1,24 @@
+import { registerComponent } from 'meteor/vulcan:core';
+import React, { useCallback } from 'react';
+import { useTracking } from "../../lib/analyticsEvents";
+
+const mouseButtonDecoding = {
+  0: 'main_button',
+  1: 'auxilliary_button',
+  2: 'secondary_button',
+}
+const AnalyticsTracker = ({eventType, eventProps, children, captureOnClick=true, captureOnMount, skip}) => {
```

Comment:
The `skip` and `captureOnClick` props are unused

Reply:
Huh? They've used in the handleClick callback.